database/raft/raft.go

```diff
@@ -201,6 +202,7 @@ func Start(laddr, dir, bootURL string, httpClient *http.Client) (*Service, error
 		Applied:         sv.state.AppliedIndex(),
 		MaxSizePerMsg:   4096,
 		MaxInflightMsgs: 256,
+		Logger:          &raft.DefaultLogger{Logger: stdlog.New(ioutil.Discard, "", 0)},
```

Comment:
It discards the coreos raft package log output. Its [default logger](https://github.com/chain/chain/blob/50dff7d598be223161a5a56299719cff95cd99ab/vendor/github.com/coreos/etcd/raft/logger.go#L47-L49) goes to stderr.

I didn't see any difference in the output in my tests, but I suppose it's possible I just didn't hit anything that causes that package to print log messages.